language-server/modules/langserver-core/src/test/resources/completion/expression_context/config/mapping_expr_ctx_config29.json

```diff
@@ -13,39 +13,6 @@
       "filterText": "readonly",
       "insertText": "readonly ",
       "insertTextFormat": "Snippet"
-    },
```

Comment:
Completion changes due to https://github.com/ballerina-platform/ballerina-lang/issues/35237

Reply:
@pcnfernando @nadeeshaan Now the representation is different from what we had earlier, can this be a breaking change for LS side?

Reply:
The issue here is completion depends on the semantic api which relies on the erroneous behaviour. Ideally completions should work the same. We'll have to handle it in the semantic api accordingly. This might be a breaking change if we consider semantic api changes as breaking.

Reply:
This is a breaking change IMHO, given the internal behaviour change in Semantic API